agr_java_core/src/main/java/org/alliancegenome/core/variant/service/AlleleVariantIndexService.java

```diff
@@ -304,8 +294,7 @@ public BoolQueryBuilder buildBoolQuery(String id, Pagination pagination) {
 					}
 
 					if (e.getKey().toString().equalsIgnoreCase("allele_category")) {
-						if (e.getValue().toString().split("\\|").length < 3)
-							queryBuilder.filter(QueryBuilders.termsQuery("alterationType.keyword", e.getValue().toString().split("\\|")));
+						queryBuilder.filter(QueryBuilders.termsQuery("alterationType.keyword", e.getValue().toString().split("\\|")));
```

Comment:
Was alteration type a two digit code before? Seems weird if it was.

Reply:
I cannot remember but it looks like the list of values used to be less than 3.